windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -76,54 +76,4 @@ Windows nodes are now fully integrated with most of the monitoring capabilities
 
-* If you have a cluster with two Windows nodes, and you create a web server deployment with two replicas, each pod lands on a Windows compute node. In this scenario, if you create a `Service` object with type `LoadBalancer`, communication with the load balancer endpoint is not reliable. To mitigate this issue for clusters installed on AWS or Azure, you must use Windows Server 2019 version 10.0.17763.1457 or earlier. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1905950[*BZ#1905950*])
-+
-To pick the correct image for the `MachineSet` object, follow the instructions based on your cloud provider:
-+
---
-*AWS*
-
-. Run the following command to list AWS image info:
-+
-[source,terminal]
-----
-$ aws ec2 describe-images \
-    --filters Name=name,Values=Windows_Server-2019-English-Full-ContainersLatest-2020.09.09
-    --region <region> \// <1>
-    --query 'Images[*].[ImageId]' \
-    --output=json | jq .[0][0]
-----
-<1> Specify the region your cluster is using.
-
-. Find the AMI ID for a supported Windows image.
-
---
-+
---
-*Azure*
-
-. Run the following command to list Azure image info:
-+
-[source,terminal]
-----
-$ az vm image list --all --location <location> \// <1>
-    --publisher MicrosoftWindowsServer \
-    --offer WindowsServer \
-    --sku 2019-Datacenter-with-Containers \
-    --query "[?contains(version, '17763.1457.2009030514')]"
-----
-<1> Specify the location your cluster is using.
-
-. Find a `WindowsServer` image offering that defines the `2019-Datacenter-with-Containers` SKU with version `17763.1457.2009030514` or earlier.
-+
-.Example output
-----
-    "offer": "WindowsServer",
-    "publisher": "MicrosoftWindowsServer",
-    "sku": "2019-Datacenter-with-Containers",
-    "urn": "MicrosoftWindowsServer:WindowsServer:2019-Datacenter-with-Containers:17763.1457.2009030514",
-    "version": "17763.1457.2009030514"
-----
---
-
-* There is currently an issue in Windows Server 2019 versions released after version `10.0.17763.1457` where Windows workloads behind a load balancer are unreachable for clusters installed on AWS and Azure. You must use Windows Server 2019 version `10.0.17763.1457` or earlier to work around this issue; however, these earlier images are no longer available for AWS. This image version unavailability prevents the ability to run Windows workloads behind a load balancer on clusters installed on AWS at this time. See the link:https://github.com/microsoft/Windows-Containers/issues/78[Microsoft Windows Containers issue] for more details.
+* The kube-proxy service terminates unexpectedly after the load balancer is created if you create the load balancer after the Windows pods begin running. To avoid this issue, you must create the load balancer before the Windows pods' status transitions to `Running`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939968[*BZ#1939968*])
```

Comment:
s/pods'/pods/
ISG advises to avoid parentheses by dropping or writing around where possible to avoid translation issues. It's fine to drop the `'` in this case.